### ISSUE TITLE
A problem about the the orthogonal constraint of TransH

### ISSUE THREAD

**Issue by SunYatong** [thunlp/TensorFlow-TransX#30]
Hi, to begin with, thanks a lot for your great efforts of developing this project.

I have a problem with the loss function of TransH,

https://github.com/thunlp/TensorFlow-TransX/blob/1f2ce7eaf20d669534258b840fa5ba9a42329d1a/transH.py#L83

which seems to ignore the orthogonal constraint of **e**_{r} and **w**_{r} in the original paper, or I missed its implementation?

Looking forward to your reply. Thanks in advance for considering my issue.

**Comment by THUCSTHanxu13** [comment]
The effect of the orthogonal constraint is not significant in our implementation, we thus ignore this part. I think you can add this part and have a try.
